content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md

```diff
@@ -0,0 +1,184 @@
+  - Check installed version ```kustomize version```
+- Install and configure the [Azure Command Line Interface (Az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  - Log in with ```az login```
+- (Optional) Install Docker
+  - For Windows and WSL: [Guide](https://docs.docker.com/docker-for-windows/wsl/)
+  - For other OS: [Docker Desktop](https://docs.docker.com/docker-hub/)
+
+You do not need to have an existing Azure Resource Group or Cluster for AKS (Azure Kubernetes Service). You can create a cluster in the deployment process.
+
+## Deployment
+
+The deployment process is split into two ways:
+
+* **Install with a single command** - Entire Kubeflow is installed using a single command. You only need to run `while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done` if you want
+  to build all the components of Kubeflow [Guide](https://github.com/kubeflow/manifests#install-with-a-single-command)
+* **Install individual components** - Installs each individuals components of Kubeflow seperately. [Guide](https://github.com/kubeflow/manifests#install-individual-components)
+
+## Azure setup
+
+### Azure Login
+
+To log into Azure from the command line interface, run the following commands
+
+  ```
+  az login
+  az account set --subscription <NAME OR ID OF SUBSCRIPTION>
+  ```
+
+### Initial cluster setup for new cluster
+
+Create a resource group:
+
+  ```
+  az group create -n <RESOURCE_GROUP_NAME> -l <LOCATION>
+  ```
+
+Example variables:
+
+- `RESOURCE_GROUP_NAME=KubeTest`
+- `LOCATION=westus`
+
+Create a specifically defined cluster:
+  
+  ```
+  az aks create -g <RESOURCE_GROUP_NAME> -n <NAME> -s <AGENT_SIZE> -c <AGENT_COUNT> -l <LOCATION> --generate-ssh-keys
+  ```
+
+Example variables:
+
+- `NAME=KubeTestCluster`
+- `AGENT_SIZE=Standard_D4s_v3`
+- `AGENT_COUNT=3`
+- `RESOURCE_GROUP_NAME=KubeTest`
+
+**NOTE**:  If you are using a GPU based AKS cluster (For example: AGENT_SIZE=Standard_NC6), you also need to [install the NVidia drivers](https://docs.microsoft.com/azure/aks/gpu-cluster#install-nvidia-drivers) on the cluster nodes before you can use GPUs with Kubeflow.
+
+## Kubeflow installation
+
+After creating resource group and AKS, run the following commands
+
+1. Create user credentials. You only need to run this command once, it will update kubeconfig file to current cluster that you have setup .
+
+    ```
+    az aks get-credentials --resource-group <RESOURCE_GROUP_NAME> --name <CLUSTER_NAME> --admin
+    ```
+
+1. Download the kubectl v1.20.5 release from the
+  [kubectl page](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux).
+
+1. Check installed version:
+
+    ```
+    kubectl version --client
+    ```
+
+1. Run the following commands to set up and deploy Kubeflow. 
+    ```
+    git clone https://github.com/kubeflow/manifests
```

Comment:
I guess this is for kubeflow v1.5, do you want to pin the branch here? for kubeflow 1.5, there is `v1.5-branch` branch.

Reply:
It is for Kubeflow 1.5 how do I pin the branch here ?

Reply:
@yhwang I checked, this PR is a website-documentation update and it should not be dependent on a branch, as its pushed to Kubeflow Website.

Reply:
@yhwang I have created PR for branch 1.5 here: https://github.com/kubeflow/website/pull/3284 please review